{
  "schema_version": "1.3.1",
  "id": "GO-2023-1985",
  "modified": "0001-01-01T00:00:00Z",
  "published": "0001-01-01T00:00:00Z",
  "aliases": [
    "CVE-2015-7561",
    "GHSA-2h9c-34v6-3qmr"
  ],
  "summary": "Kubernetes in OpenShift3 Access Control Misconfiguration in k8s.io/kubernetes",
  "details": "Kubernetes in OpenShift3 Access Control Misconfiguration in k8s.io/kubernetes",
  "affected": [
    {
      "package": {
        "name": "k8s.io/kubernetes",
        "ecosystem": "Go"
      },
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "1.2.0-alpha.6"
            }
          ]
        }
      ],
      "ecosystem_specific": {}
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://github.com/advisories/GHSA-2h9c-34v6-3qmr"
    },
    {
      "type": "WEB",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1291963"
    },
    {
      "type": "WEB",
      "url": "https://github.com/kubernetes/kubernetes/commit/e185b1028ac8459f7b451e1115399192e96f6ee9"
    },
    {
      "type": "WEB",
      "url": "https://github.com/kubernetes/kubernetes/pull/18909"
    }
  ],
  "database_specific": {
    "url": "https://pkg.go.dev/vuln/GO-2023-1985",
    "review_status": "REVIEWED"
  }
}